dvc/command/init.py

```diff
@@ -56,4 +59,13 @@ def add_parser(subparsers, parent_parser):
             "This operation removes local cache."
         ),
     )
+    init_parser.add_argument(
+        "--subrepo",
+        action="store_true",
+        default=False,
+        help=(
+            "Use this flag if created repository is below Git repository"
+            " in tree."
```

Comment:
Please move the space above, for consistency with other messages.

Reply:
@jorgeorpinel Could you please take a look at this help message? :pray:

Reply:
Maybe `Necessary for running this command in a subdirectory of an existing DVC repository.`

Reply:
Maybe `Necessary for running this command inside a subdirectory of a parent DVC repository.`

Will check other strings... Thanks